mindmeld/components/entity_resolver.py

```diff
@@ -139,27 +139,22 @@ def create_resolver(cls, app_path, entity_type, **kwargs):
             es_client (Elasticsearch): The Elasticsearch client.
         """
 
-        er_config = (
-            kwargs.pop("config", {}) or
-            get_classifier_config("entity_resolution", app_path=app_path)
-        )
+        er_config = config or get_classifier_config("entity_resolution", app_path=app_path)
         er_config = cls._correct_deprecated_er_config(er_config)
 
         resolver_type = er_config["model_settings"]["resolver_type"]
         cls._validate_resolver_type(resolver_type)
 
-        resource_loader = kwargs.pop(
-            "resource_loader",
-            ResourceLoader.create_resource_loader(app_path=app_path)
```

Comment:
This line was creating extra ResourceLoaders to pass in for the default value that were then discarded if not needed.